src/config.ts

```diff
@@ -43,3 +48,34 @@ export const showHomeServerDetail = loginScreen?.showHomeServerInfo ?? true;
 
 // Default footer links
 export const showDefaultFooterLinks = loginScreen?.showMatrixDefaultFooterLinks ?? true;
+
+// show/hide liberate your communication text
+export const showLiberateYourCommunicationText = authenticatedHomeScreen?.showLiberateYourCommunicationText ?? true;
+
+//show/hide welcome to {brand} text
+export const showWelcomeToElementText = authenticatedHomeScreen?.showWelcomeToElementText ?? true;
+
+// show primary logo on logged in screen
+// Due to positioning and etc this switch would make it easy to have that logo configured.
+// it is to control logo that element provides
+export const showPrimaryLogoOnLoginScreen = logoConfig?.showPrimaryLogoOnLoginScreen ?? true;
```

Comment:
I can't see where the showPrimaryLogo* config items are still used?  Please review all the config items in this file and any still used please ensure their description is added to feature-communicate-list.md

Reply:
done